### PR TITLE
projectile homework

### PR DESCRIPTION
**Pull request by Moussiaa**

### CODE REVIEW

File: src/main/java/adry/projectile/Projectile.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class Projectile {
+
+    private final double angle;
+    private final double radians;
+    private final double velocity;
+    private double seconds;
+
+
+
+
```
Comment (andrewoid) [inline src/main/java/adry/projectile/Projectile.java]: Extra space

File: src/main/java/adry/projectile/Projectile.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public double getY() {
+        return Math.sin(radians) * velocity * seconds - .5 * 9.8 * seconds * seconds;
+    }
+
+    //when projectile is at its highest
+    public double getApex()
+    {
+        double v = velocity * Math.sin(radians);
+        return v / 9.8;
```
Comment (andrewoid) [inline src/main/java/adry/projectile/Projectile.java]: Put this on one line

File: src/main/java/adry/projectile/Projectile.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    private final double angle;
+    private final double radians;
+    private final double velocity;
+    private double seconds;
+
+
+
+
+    // construct the class
+    public Projectile(double angle, double velocity, double seconds) {
```
Comment (andrewoid) [inline src/main/java/adry/projectile/Projectile.java:11]: Seconds shouldn't be part of the constructor.

Create a setSeconds method

File: src/main/java/adry/projectile/Projectile.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    public double getX() {
+        return Math.cos(radians) * velocity * seconds;
+    }
+
+    public double getY() {
+        return Math.sin(radians) * velocity * seconds - .5 * 9.8 * seconds * seconds;
+    }
+
+    // highest Y value of the Projectile
+    public double getPeakY() { return (velocity * (Math.sin(angle) * Math.sin(angle)) / (2 * 9.8)); }
```
Comment (andrewoid) [inline src/main/java/adry/projectile/Projectile.java]: You're missing a test for this method.

File: src/test/java/adry/projectile/ProjectileTest.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Test
+    public void getApex() {
+        // given
+        Projectile projectile = new Projectile(31.00, 20.00, 2.70);
+        // when
+        double actual = projectile.getApex();
+        // then
+        assertEquals(1.05109811206, actual, 0.01);
```
Comment (andrewoid) [inline src/test/java/adry/projectile/ProjectileTest.java]: 1.05 is enough

**Comment by andrewoid** [comment]
Not sure why you hit the merge button. I did not approve your PR yet. We are not finished.

**Comment by Moussiaa** [comment]
Oh no I just downloaded the app and followed the prompt to merge😭 is this
something I can undo?

**Comment by andrewoid** [comment]
> Oh no I just downloaded the app and followed the prompt to merge😭 is this
> something I can undo?

Downloaded what app?

**Comment by Moussiaa** [comment]
GitHub on my phone

**Comment by andrewoid** [comment]
> GitHub on my phone

I think you can reopen the PR if you are in a browser. Let's talk tomorrow. I left another comment on your PR, you can still fix it on the branch on your computer.

**Comment by Moussiaa** [comment]
Ok, thanks.

**Pull request merged**